check_version.sh

```diff
@@ -0,0 +1,10 @@
+changelog_version=$(head -n 1 CHANGELOG.md)
+maven_version=$(mvn help:evaluate -Dexpression=project.version | grep -v -e "^\\[")
+
```

Comment:
we should probably add a check that it is a valid version string, too, else the empty string would match.

Reply:
I think we can assume that these won't both break at the same time, though if anyone wants to suggest a simple change that would implement this request then I'm happy to apply.